### PR TITLE
Fix and update some tests

### PR DESCRIPTION
**Pull request by tom-parker**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -59,3 +59,5 @@ typings/
 
 # other
 app/*.properties
+
+.editorconfig
```
Comment (leafrogers) [inline .gitignore]: Why not add it?

**Review by leafrogers — COMMENTED** [review]
Would be good to set up CI for this app

File: .gitignore (hunk 1 above)
Comment (tom-parker) [reply .gitignore]: Have you got an example one you've added elsewhere?

Comment (leafrogers) [reply .gitignore]: Recently added one to [simple-email-service](https://github.com/Financial-Times/simple-email-service/blob/master/.editorconfig)

**Comment by leafrogers** [comment]
Ace ✌️

**Pull request merged**